{{cookiecutter.project_name}}/.mlops-setup-scripts/README.md

```diff
@@ -140,10 +144,20 @@ be sure to generate a token with "Repo" scope. If you have SSO enabled with your
 {% elif cookiecutter.cicd_platform == "azureDevOpsServices" -%}
 This token is used to fetch ML code from the current repo to run on Databricks for CI/CD (e.g. to check out code from a PR branch and run it
 during CI/CD). You can generate a PAT token for Azure DevOps by following the steps described [here](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows).
-
-### Grant version control permissions to CI/CD
-The provided CI/CD workflows attempt to run `git` commands to commit and modify files. To ensure the workflows can work properly, [grant version control permissions](https://learn.microsoft.com/en-us/azure/devops/pipelines/scripts/git-commands?view=azure-devops&tabs=yaml#grant-version-control-permissions-to-the-build-service) within your hosted Azure DevOps repo.
-The user `{{cookiecutter.project_name}} Build Service (<your-username>)` should be granted the following:
+Ensure your PAT (at a minimum) has the following permissions:
+- **Variable Groups**: Read, Create, & Manage
+- **Build**: Read & execute
+- **Project and Team**: Read
+- **Token Administration**: Read & manage
+- **Tokens**: Read & manage
+- **Work Items**: Read
+
+### Update permissions for the build service
+In our CI/CD workflow, upon successfully merging a PR with changes to `databricks-config/**` the CD step will trigger the
+Terraform deploy stage of our pipeline. As part of this pipeline we run `git` commands to commit and modify Terraform output files. 
+To enable this workflow we must update the permissions of our build service. Within your **Project Settings**, select **Repostiories**.
+Go to the name of your repository and select **Security**. For the user `{{cookiecutter.project_name}} Build Service (<your-username>)` grant the following:
+- **Bypass policies when completing pull requests**: Allow
```

Comment:
nit for consistency and clarity: please use "you" as opposed to "we" for actions that the user is supposed to perform.

Reply:
done!